### ISSUE TITLE
Feature Request:  Recent Folders

### ISSUE THREAD

**Issue by wtfsven** [adobe/brackets#10668]
I'd love to have a "Open Recent Folder..." option in the File menu.  I tend to switch between projects a lot and it would be nice to have a 2-click way of doing that.

**Comment by sbruchmann** [comment]
Hello @wtfsven,

I’m not quite sure if you mean this, but you can select your recent projects (folders, respectively) by clicking on the current project name in the sidebar:

![screen shot 2015-03-01 at 10 35 26 pm](https://cloud.githubusercontent.com/assets/141435/6432968/87d2bdd8-c063-11e4-8e4c-59c6f8f1ec8f.png)

**Comment by wtfsven** [comment]
That's exactly what I mean, actually.  Go ahead and close this, I guess.  :P

--- Original Message ---

**Comment by prafulVaishnav** [comment]
Thanks @sbruchmann This is exactly two click way of switching :) Closing this.

**Comment by peterflynn** [comment]
@wtfsven You can also do this via the keyboard using Ctrl-Alt-R (or Cmd-Alt-R on Mac).

**Comment by redmunds** [comment]
FYI, I'd like to see that shortcut displayed somewhere because I can never remember it. I think the Recent Project list should also be somewhere in the main menu and the shortcut can be shown there. If anyone agrees I can create an issue.

**Comment by peterflynn** [comment]
I'd love to see it in the menu, but that's still blocked on [submenu support](https://trello.com/c/EwLGRkYe/541-native-submenus)...

In the meantime, maybe just putting the shortcut in the tooltip of the project dropdown would help?  We put shortcuts in tooltips elsewhere, e.g. the Find bar next/prev buttons.

**Comment by wtfsven** [comment]
Well now I understand the design decision.  A not-so-elegant work-around could be "Recent Projects..." which opens a folder dialog to folder with symbolic links to previous folders generated as they're opened, but that may be overkill.

--- Original Message ---
